service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOController.java

```diff
@@ -52,4 +68,21 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
             .data(caseDetails.getData())
             .build();
     }
+
+    private CaseDetails setCustomDirectionDropdownKeys(CaseDetails caseDetails) {
+        caseDetails.getData().put("otherPartiesDropdownKeyCMO",
+            draftCMOService.createOtherPartiesAssigneeDropdownKey(caseDetails));
+
+        caseDetails.getData().put("parentsAndRespondentsDropdownKeyCMO",
+            draftCMOService.createParentsAndRespondentAssigneeDropdownKey(caseDetails));
+
+        return caseDetails;
+    }
+
+    private CaseDetails removeExistingDirectionsFromCaseDetails(CaseDetails caseDetails) {
```

Comment:
Just thinking do we need `FromCaseDetails` in method name as it seems bit obvious from Method signature. I think `removeExistingDirections` should be fine.